elastic/kibana
# [ResponseOps][Connectors] Fix ServiceNow ITSM connector for security detection rules

## Summary

Fixes https://github.com/elastic/kibana/issues/189286

<img width="669" alt="image" src="https://github.com/user-attachments/assets/4101f136-5efc-461f-8a49-c2b232d8b16a">

**How to test:**
- Add service now itsm connector to security solution's detection rules. verify it works as expected.
- Add service now itsm connector to observability / stack rules. verify it works as expected.
- Verify connector works as expected in test tab

### Checklist
